content/desktop/hardened-desktop/_index.md

```diff
@@ -45,17 +49,20 @@ It is for security conscious organizations who:
 ### What does Hardened Docker Desktop include?
 
 It includes:
+
+{{< grid >}}
```

Comment:
Sure can do, but let me say: as a reader, I found it ** much clearer ** to read the grid compared to bulleted text. It immediately tells me what are the components of the feature, and I can then click on each. 

Thus I would encourage moving the grids up higher in the other pages to increase their visibility and make it easier for readers, rather than leaving them at the bottom where they are less helpful / visible.

Reply:
OK thanks, I've moved the grid to where it was. I think if we make the page more concise (follow-up PR), it would help as right now it's a bit wordy for an intro page.